Fixes to SDXL accuracy output

arjunsuresh opened the pull request.

mrmhodak commented:
@pgmpablo157321 to review

pgmpablo157321 commented:
@arjunsuresh Regarding the `format.yml` action, it isn't working on forked pull request. But I see that this is removing it completely. Should we fix it? Otherwise LGTM, I am ok merging this and opening an issue for this

arjunsuresh commented:
Thank you @pgmpablo157321 for reviewing. I just added the GH action file for the push (instead of on PR) - I believe I had deleted it by mistake earlier.
